.circleci/old-versions.py

```diff
@@ -0,0 +1,29 @@
+"""
+Version helper.
+
+Checks the directory 'build_dir' and return list of last 7 versions.
+[dev, tag, tag-1, tag-2,..., tag-5]
+
+Then updates file version-helper.js, which takes care of injecting versions
```

Comment:
Is `version-helper.js` dynamically generated?

Reply:
yes file `version-helper.js` is overwritten on each run